front/components/assistant/conversation/Conversation.tsx

```diff
@@ -123,8 +124,16 @@ export default function Conversation({
           case "user_message":
             return (
               <div key={`message-id-${m.sId}`} className="bg-structure-50 py-6">
-                <div className="mx-auto flex max-w-4xl gap-4">
-                  <UserMessage message={m} />
+                <div className="mx-auto flex max-w-4xl flex-col gap-4">
+                  <UserMessage message={m}>
```

Comment:
Let's push the conversation inside UserMessage (I will need it for citations in RenderMarkdown) and remove children here :+1:

Reply:
s/conversation/AgentSuggestion/ ?

Reply:
No the idea is to push the ConversationType into the UserMessage so that you don't need children and implement the AgentSuggestion there so yeah push both the conversation state and the AgentSuggestion components